pulsar-client-tools/src/main/java/com/yahoo/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -34,7 +34,7 @@
     private boolean help;
 
     PulsarAdminTool(Properties properties) throws Exception {
-        String serviceUrl = properties.getProperty("serviceUrl");
+        String serviceUrl = properties.getProperty("webServiceUrl");
```

Comment:
Can you try to fallback to the `serviceUrl` if `webServiceUrl` is not present? That way we preserve compatibility